sdk/src/main/java/com/bugsnag/android/Report.java

```diff
@@ -14,9 +14,6 @@
  */
 public class Report implements JsonStream.Streamable {
 
-    @Nullable
-    private final File errorFile;
-
     @Nullable
```

Comment:
Wouldn't that be a breaking change? Any existing reference to `report.error?` would need to remove the `?`, right?

Reply:
It would be marked as redundant but isn't a breaking change. Switching from NonNull to Nullable *would* be a breaking change however, so we'd need to be sure about this.

Reply:
Reverting this bit to only deserialize reports if there are callbacks to be run.